app/views/gobierto_plans/plan_types/_planification-content.html.erb

```diff
@@ -111,49 +111,74 @@
+                  <div class="optional-obj" v-for="(_value, _key) in value">
+                    <div class="optional-title">{{ optionKeys[_key] | translate }}</div>
+                    <div class="optional-desc">
+                      <template v-if="typeOf(_value) === 'string'">
+                        {{ _value }}
+                      </template>
+                      <template v-if="typeOf(_value) === 'array'">
+                        <div v-for="(_val, _key) in _value">{{ _val }}</div>
+                      </template>
+                    </div>
                   </div>
                 </div>
-              </div>
-            </template>
-          </div>
-        </div>
-        <div class="project-mandatory">
-          <div class="row">
-            <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.progress") %></div>
-            <div class="mandatory-desc mandatory-desc--large">{{ activeNode.attributes.progress | percent }}</div>
-          </div>
-          <div class="row">
-            <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.starts") %></div>
-            <div class="mandatory-desc">{{ activeNode.attributes.starts_at | date }}</div>
-          </div>
-          <div class="row">
-            <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.ends") %></div>
-            <div class="mandatory-desc">{{ activeNode.attributes.ends_at | date }}</div>
+              </template>
+            </div>
           </div>
-          <div class="row">
-            <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.status") %></div>
-            <div class="mandatory-desc">{{ activeNode.attributes.status | translate }}</div>
+          <div class="project-mandatory">
+            <div class="row">
+              <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.progress") %></div>
+              <div class="mandatory-desc mandatory-desc--large">{{ activeNode.attributes.progress | percent }}</div>
+            </div>
+            <div class="row">
+              <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.starts") %></div>
+              <div class="mandatory-desc">{{ activeNode.attributes.starts_at | date }}</div>
+            </div>
+            <div class="row">
+              <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.ends") %></div>
+              <div class="mandatory-desc">{{ activeNode.attributes.ends_at | date }}</div>
+            </div>
+            <div class="row">
+              <div class="mandatory-title"><%= t("gobierto_plans.plan_types.show.status") %></div>
+              <div class="mandatory-desc">{{ activeNode.attributes.status | translate }}</div>
+            </div>
           </div>
         </div>
+
+        <!-- Rails must add the markup if plugin is available -->
+        <% availablePlugins = ['plugin1', 'plugin2'] %>
```

Comment:
cc/ @ferblape @amiedes @entantoencuanto 

First off, ruby gets all available plugins from configuration. 
1. Add the markup as following:  `<div id="WHATEVER_ELEMENT_ID"></div>`
2. Call the controller function `add` to insert the plugin as: `window.GobiertoPlans.tablerow_controller.add("FETCHED KEY", "WHATEVER_ELEMENT_ID")` <sup>1</sup>
3. Start the listeners as: `window.GobiertoPlans.tablerow_controller.show()`. That could be inserted before the plugins markup, it doesn't matter.

<sup>1</sup> Fetched key is the name of the children key of the `plugins_data` object. [Check this](https://github.com/PopulateTools/issues/issues/689#issuecomment-505869087)